src/main/resources/db/migration/V1_2__authorised_by_size_change.sql

```diff
@@ -0,0 +1,2 @@
+alter table non_association
+    alter column authorised_by type varchar(60);
```

Comment:
We should do this on all username/staff name fields if we do it here, no?

Reply:
Nope! @xoen see below